### DISCUSSION TITLE
Welcome to smart-thermostat Discussions!

### DISCUSSION THREAD

**Discussion by smeisner** [smeisner/smart-thermostat#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by corentin-ltc** [comment]
Hey, very good project!  I like the innovation of having it connected to the forecast. I'm a CP student and would like to learn electronics and start with your project to make it my thermostat at home. 
So I have a few questions:
-Atm I have the sensor LD241, the ESP32, the power supply LM2576HVT-5 and the MSP3218 touchscreen. Do I need to buy some other components ?
Also, if its possible,  I'd like to see pictures of inside your thermostat to make it more clear for me about the hardware if you don't mind.
As far as i read, there wasnt a mention of the hot water management, this would be great !

Thats it for my impressions, thank you very much for this awesome projects!

**Comment by smeisner** [comment]
@corentin-ltc Hi! This project is half embedded code and, very importantly, the custom PCB I designed (also in this Github project). You could "cobble" together the components (similar to what I did in the very beginning), but you will be diverging from what we are trying to do here.

For pics, you can load up KiCad and load my PCB project. Then use the 3D viewer to see a rendered 3D model.

No, no mention of hot water since this is an HVAC thermostat. Not a hot water controller.
